examples/altair_charts/app.py

```diff
@@ -0,0 +1,37 @@
+from fh_altair import altair2fasthml, altair_headers
+from fasthtml.common import * 
+import numpy as np
+import pandas as pd
+import altair as alt
+
+app, rt = fast_app(hdrs=altair_headers)  
+
+count = 0
+plotdata = []
+
+def generate_chart():
+    pltr = pd.DataFrame({'y': plotdata, 'x': range(len(plotdata))})
```

Comment:
With this always incrementing and not resetting, it may be difficult to see the chart update once it gets too large (either due to decent traffic or just because of long time between deploys).

I think we should either not have it global variable, or maybe plot `plotdata[-50:]` or something if there's a lot of `plotdata`